frontend/src/metabase/collections/components/MoveCollectionModal/MoveCollectionModal.tsx

```diff
@@ -23,7 +24,7 @@ const MoveCollectionModal = ({
 
   return (
     <CollectionMoveModal
-      title={`Move ${collection.name}?`}
+      title={t`Move "${collection.name}"?`}
```

Comment:
![Screenshot 2023-08-11 at 8 02 17 PM](https://github.com/metabase/metabase/assets/1937582/bba430b5-c2d4-4c2a-80b1-8b76404adbc8)

Modify the string to contain double quotes as well so we reuse an existing translation from

https://github.com/metabase/metabase/blob/73cb16ee60c2d1742691fc0df5f7b05d9c24654a/frontend/src/metabase/collections/components/BulkActions.jsx#L97